feature-utils/poly-look/src/react-components/buttons/iconButton.css

```diff
@@ -0,0 +1,16 @@
+.icon-button {
```

Comment:
See my other comment - this stuff should be prefixed to avoid conflicts with other CSS classes, i.e. `.poly-icon-button`.

Reply:
I've prefixed the class, but kept the component name.